### PR TITLE
Allow relative all paths used in CLI/API

### PR DESCRIPTION
**Pull request by mboudreau**

### CODE REVIEW

**Comment by mboudreau** [comment]
@mefellows I'm waiting with baited breath ;)

**Comment by mefellows** [comment]
Looking at this shortly, Michel.

File: src/publisher.spec.ts [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 import * as http from "http";
 
+const rimraf = require("rimraf");
+const mkdirp = require("mkdirp");
 const expect = chai.expect;
 chai.use(chaiAsPromised);
-const currentDir = (process && process.mainModule) ? process.mainModule.filename : "";
 
-describe("Publish Spec", () => {
+describe.only("Publish Spec", () => {
```
Comment (mefellows) [inline src/publisher.spec.ts]: Please remove

File: src/server.spec.ts [hunk 2]
```diff
@@ -8,15 +8,38 @@ import path = require("path");
...
 import _ = require("underscore");
 
+const mkdirp = require("mkdirp");
+const rimraf = require("rimraf");
+
 chai.use(chaiAsPromised);
 const expect = chai.expect;
 
-describe("Server Spec", () => {
+describe.only("Server Spec", () => {
```
Comment (mefellows) [inline src/server.spec.ts]: Please remove

File: src/message.spec.ts [hunk 3]
```diff
@@ -1,72 +1,127 @@
...
-const path = require("path");
+
+const mkdirp = require("mkdirp");
+const rimraf = require("rimraf");
 const expect = chai.expect;
 chai.use(chaiAsPromised);
 
-describe("Message Spec", () => {
-	const currentDir = (process && process.mainModule) ? path.dirname(process.mainModule.filename) : "";
+describe.only("Message Spec", () => {
```
Comment (mefellows) [inline src/message.spec.ts]: Please remove

File: src/publisher.ts [hunk 4]
```diff
@@ -34,6 +36,16 @@ export class Publisher {
 
 		if (options.pactFilesOrDirs) {
 			checkTypes.assert.array.of.string(options.pactFilesOrDirs);
+			for(let i = 0, len = options.pactFilesOrDirs.length; i<len; i++) {
```
Comment (mefellows) [inline src/publisher.ts]: Any reason this couldn't be something like `options.pactFilesOrDirs.forEach((item) => { ... } )` ?

**Comment by mefellows** [comment]
🎉

**Pull request merged**